src/ProjectNeon/Assets/Features/Battle/Debug/DebugDrawCard.cs

```diff
@@ -9,4 +10,9 @@ public void DrawOneCard()
     {
         Hand.PutOnBottom(Deck.DrawOneCard());
     }
+
+    public static void Log(EffectData effectData)
```

Comment:
What is this for?

Reply:
@SilasReinagel More automatic implementations. Looks like I'm making mistakes with VisualStudio shortcuts